config/conanfile.py

```diff
@@ -12,17 +12,12 @@ class KatanaConan(ConanFile):
         "backward-cpp/1.5",
         "benchmark/1.5.0",
         "boost/1.71.0",
-        "brotli/1.0.9",
         "date/2.4.1",
         "eigen/3.3.7",
         "fmt/6.2.1",
         "libcurl/7.74.0",
-        "lz4/1.9.3",
```

Comment:
I guess the `arrow` conan package will add these anyway if `arrow:with_foo` is True?

Reply:
Yes. You removed them. ;-) https://github.com/KatanaGraph/katana-enterprise/commit/a5810eeaedb8bcfb05f6c00a0fbcc4796124ce44#diff-ce048b1297f1f5f485c949613942256e74b195a6e05dc761d179a2857382e38b